packages/scratch-gui/src/lib/account-menu-options.ts

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+
+export interface AccountMenuOptions {
+    canHaveSession: boolean;
+
+    canRegister?: boolean;
+    canLogin?: boolean;
+    canLogout?: boolean;
+
+    avatarUrl?: string;
+    myStuffUrl?: string;
+    profileUrl?: string;
+    myClassesUrl?: string;
+    myClassUrl?: string;
+    accountSettingsUrl?: string;
+}
+
+export const AccountMenuOptionsPropTypes = PropTypes.shape({
+    canHaveSession: PropTypes.bool.isRequired,
```

Comment:
In what scenarios would `canHaveSession` be false? Is that for, say, Scratch Lab?

Reply:
I believe so, yes. If this is false then neither "Sign up / sign in", nor "<username> / log out" is shown

Reply:
This was originally called "sessionExists" but it seems like the session in www always exists, even if you're not logged in, so I renamed it here